src/accounting.py

```diff
@@ -585,13 +603,127 @@ def create_initial_ledger(unannotated_ledger_path, result_path, leading_zero_cou
     with open(result_path, 'w') as f:
         f.writelines(line + '\n' for line in lines)
 
+
+class WealthTaxBracket:
+    """Tax Bracket Object used to represent wealth tax brackets so they can be easily modified"""
+
+    def __init__(self, start, end, rate, exempt_prefixes=None):
+        if exempt_prefixes is None:
+            exempt_prefixes = ['&', '@'] # default prefixes for government and non-profits
+        self.start = start
+        self.end = end
+        self.tax_rate = rate
+        self.exempt_prefixes = exempt_prefixes
+
+    def set_rate(self, rate):
+        # sets tax rate
+        self.tax_rate = rate
+
+    def add_exempt_prefix(self, prefix):
+        # adds exempt prefixes
+        self.exempt_prefixes.append(prefix)
+
+    def get_rate(self):
+        # returns tax rate for bracket object
+        return self.tax_rate
+
+    def set_end(self, end):
+        # sets the end of the starting
+        self.end = end
+
+    def set_start(self, start):
+        self.start = start
+
+    def get_start(self):
+        return self.start
+
+    def get_end(self):
+        return self.end
+
+    def get_tax(self, account):
+        bal = account.get_balance()
+        if bal < self.start:
+            return 0
+        elif self.end is None or bal <= self.end:
+            tax_amount = round(((bal - self.start) / 100) * self.tax_rate)
```

Comment:
Do we still need to round now that we have fractions?

Reply:
no we don't when I wrote it we did tho

Reply:
Can you get rid of the rounding then?

Reply:
actually in simdem law it legally has to be rounded to the nearest whole number

Reply:
smh that's a misfeature but okay